ui/lib/core/addon/templates/components/replication-secondary-card.hbs

```diff
@@ -0,0 +1,65 @@
+<div class="selectable-card is-rounded card-container">
+  <h3 class="title is-4 grid-item-title card-title">{{title}}</h3>
+  <div class="grid-item-state">
+    {{#if property_3}}
```

Comment:
hmm, there are a lot of conditionals in this template that depend on `property_3`. from what i can tell it seems like `property_3` will always be `Delta` and it will only ever appear on the WALs card.  have you considered ways to simplify this template? two ideas that i think are worth exploring are:
1. Changing the API of `ReplicationSecondaryCard` to be rendered like so:
```
<Dashboard.card @title="States" @data={{model}} />
<Dashboard.card @title="Last WAL Entry" @data={{model}} />
```
And having the `replication-secondary-card.js` handle the logic of determining what description and metrics to render and which template to use.

2. Allowing the `ReplicationSecondaryCard` to yield a block, giving you more flexibility with the inner contents, i.e.:
```
<Dashboard.card @title="States" @data={{model}}>
  <div class="">
... states markup
  </div>
</Dashboard.card>

<Dashboard.card @title="States" @data={{model}}>
  <div class="">
... Last WAL markup
  </div>
</Dashboard.card>
```

what i like about #1 is that it supports the point you made earlier about being able to easily see inside `details.hbs` that there are 2 cards rendered with a title of `State` and `Last WAL`. like you mentioned, it's nice to not obfuscate too much. however this gives us more flexibility with the number of metrics displayed on each card without having card coded property values like `property_value3`. similarly, i think it allows us to be more strict to prevent things like accidentally passing in the `description` for Merkle Root when we're trying to render the metric for Last WAL Entry.  it also matches the API of the `ReplicationTableRows` component, which will also be rendered on this page with `@data={{model}}` (i think). 

happy to chat more about this or hear your thoughts!

Reply:
Totally agree.  And I like your approach.  I cleaned this up.